bokehjs/src/coffee/core/selection_manager.coffee

```diff
@@ -30,8 +30,7 @@ export class SelectionManager extends HasProps
 
       @source.selected = selector.indices
 
-      source.trigger('select')
-      source.trigger('select-' + renderer_view.model.id)
```

Comment:
It's not possible to trigger events like this and this this wasn't used anywhere so I removed this. It's always possible to filter `select` on its argument.